OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationOptionSpinner.java

```diff
@@ -85,8 +101,23 @@ public void onInitializeAccessibilityEvent(AccessibilityEvent event) {
         There's an open StackOverflow issue on this as well:
         https://stackoverflow.com/questions/44708495/how-to-prevent-spinner-announcement-when-initialized
          */
-        if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED) {
+        if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED
+                || event.getEventType() == AccessibilityEvent.TYPE_VIEW_FOCUSED) {
             super.onInitializeAccessibilityEvent(event);
         }
+        if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_CLICKED ||
+                event.getEventType() == AccessibilityEvent.TYPE_VIEW_FOCUSED ||
+                event.getEventType() == AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED) {
+            if (onSpinnerFocusedListener != null) {
+                onSpinnerFocusedListener.onSpinnerFocused();
+            }
+        }
+    }
+
+    public interface OnSpinnerFocusedListener {
```

Comment:
There's also a Listener related to Spinner in the `RegistrationSelectView` class by the name `OnSpinnerItemSelectedListener`.

It would be better to refactor-move the interface from `RegistrationSelectView` to here in this class making it, which seems more logical too.